packages/block-library/src/navigation-link/edit.js

```diff
@@ -164,21 +165,23 @@ function NavigationLinkEdit( {
 						] }
 					/>
 					{ isLinkOpen && (
-						<LinkControl
-							className="wp-block-navigation-link__inline-link-input"
-							value={ link }
-							onChange={ ( {
-								title: newTitle = '',
-								url: newURL = '',
-								opensInNewTab: newOpensInNewTab,
-							} = {} ) => setAttributes( {
-								title: escape( newTitle ),
-								url: newURL,
-								label: label || escape( newTitle ),
-								opensInNewTab: newOpensInNewTab,
-							} ) }
-							onClose={ () => setIsLinkOpen( false ) }
-						/>
+						<Popover>
```

Comment:
Does it not need a position here?

Reply:
It seemed to work without it, maybe it was the default, I'll check.

Reply:
I restored this